src/components/booklist.js

```diff
@@ -1,20 +1,49 @@
 import { useDispatch } from 'react-redux';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
 import { removeBook } from '../redux/books/books';
 
 const IndividualBook = (prop) => {
   const { data } = prop;
-  const [id, [{ title, category }]] = data;
+  const [id, [{
+    title,
+    category,
+    author,
+    percentage,
+    chapters,
+  }]] = data;
   const dispatch = useDispatch();
   const removeBookFromStore = () => {
     dispatch(removeBook(id));
   };
+
   return (
     <>
-      <div>
-        {title}
-        {category}
+      <div className="books-container d-flex">
+        <div className="book-item d-flex">
+          <span className="category">{category}</span>
+          <span className="title">{title}</span>
+          <span className="author">{author}</span>
+          <button className="remove-btn" type="submit" onClick={removeBookFromStore}>Remove</button>
```

Comment:
Your final design should match Zeplin's design.

- [ ] Please add the "Comment" and "Edit" buttons as well. All element in a design should be added - Features that haven’t been implemented yet should be added as a placeholder and does not need to be implemented.

Reply:
I added the `edit` and `comments` buttons. I initially didn't include it because I wanted to follow `YAGNI` principles.